calculator.rb

```diff
@@ -0,0 +1,74 @@
+puts "Welcome to the Calculator Program! You can do any of the following: "
+puts "add (+)"
+puts "subtract(-)"
+puts "multiply(*)"
+puts "divide(/)"
+puts "exponify(**)"
+
+# print "What would you like to do: "
+# choice = gets.chomp
+
+operation = ()
+# math_stuff = ["add", "subtract", "multiply", "divide", "exponify","+", "-", "*", "/", "**"]
+print "What would you like to do: "
+choice = gets.chomp
+ if ["add", "subtract", "multiply", "divide", "exponify","+", "-", "*", "/", "**"].include?(choice)
```

Comment:
This _if_ block allows the user to give a bad operation on the second try!

iridium:calculator-master devin$ ruby calculator.rb 
Welcome to the Calculator Program! You can do any of the following: 
add (+)
subtract(-)
multiply(*)
divide(/)
exponify(**)
What would you like to do: fly
That is not math. Try again.
in the sky!
Okay, what is the first number? 5
And the second number? 4
Yay! It worked!